<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioWidget</class>
 <widget class="QWidget" name="AudioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnPlay">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>337</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>337</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenFile">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>196</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>197</y>
     <width>351</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderVol">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>16</width>
     <height>131</height>
    </rect>
   </property>
   <property name="value">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderProgress">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>277</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelProgress">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>--/--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
